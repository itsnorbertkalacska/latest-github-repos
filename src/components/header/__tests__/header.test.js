import { render } from "@testing-library/react";

import { Header } from "../index";

describe("Header", () => {
  it("renders component properly", () => {
    const { container } = render(<Header />);
    // eslint-disable-next-line testing-library/no-node-access
    expect(container.firstChild).toMatchSnapshot();
  });
});
