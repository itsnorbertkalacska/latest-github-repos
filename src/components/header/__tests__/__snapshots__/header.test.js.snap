// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders component properly 1`] = `
<header
  class="sc-gsnTZi iJFFfK"
>
  <img
    alt="Discover Trending GitHub Repos logo"
    class="sc-bczRLJ cfKNvW"
    src="logo.svg"
  />
  <h1
    class="sc-dkzDqf gcBLSw"
  >
    Discover Trending GitHub Repos
  </h1>
</header>
`;
